src/components/menu/Menu.js

```diff
@@ -113,7 +114,8 @@ export class MenuWidget extends React.Component {
       save(new Blob([ text ], { type: 'text/plain' }), filename);
     }
 
-    saveString(Exporter.generateHtml(), 'ascene.html');
+    var sceneName = getSceneName(document.querySelector('a-scene'));
```

Comment:
This is something I've in the TODO, I'll open an issue to see how we could manage multiple scenes in the same page correctly.